apps/src/assetManagement/IconList.jsx

```diff
@@ -0,0 +1,84 @@
+var IconListEntry = require('./IconListEntry.jsx');
+var icons = require('./icons').aliases;
+var msg = require('../locale');
+
+/**
+ * A component for managing icons.
+ */
+module.exports = React.createClass({
+  propTypes: {
+    assetChosen: React.PropTypes.func.isRequired,
+    search: React.PropTypes.string.isRequired
+  },
+
+  getMatches: function (query) {
```

Comment:
Maybe a function level comment explaining what we expect query to look like, and what we expect to return.